docs/index.md

```diff
@@ -46,8 +51,34 @@ Hello World
 => true
 ```
 
-As you can see from log, every action execution is wrapped inside a DB transaction.
-The main difference between these methods is: `#perform!` raises exception in case of errors and `#perform` simply returns `false`
+You can use `Granite::Action.transaction` explicitly and wrap any logic in transaction:
```

Comment:
Does not `ActiveRecord::Base.transaction` work here instead?

Reply:
Discussed in private. Granite can be used without ActiveRecord at some point
Plus this solution works like a high-level interface and we can extend its behavior at any point